doc/additional_commands.rst

```diff
@@ -45,6 +45,9 @@ Universum in non-CI mode has the following differences from default mode:
     :prog: {python} -m universum
     :path: run
 
+    --filter -f : @after
```

Comment:
Can we place the text regarding filters into separate file and include it here and in the main documentation?